src/main/resources/liquibase/workbench_changelog/22_3_0.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+	<changeSet author="aldrin" id="v22.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="one_time_password"></tableExists>
+			</not>
+		</preConditions>
+		<comment>
+			Create table 'one_time_password'
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE `workbench`.`one_time_password` (
+				`id`       INT NOT NULL AUTO_INCREMENT,
+				`otp_code` INT NULL,
+				`expires`  DATETIME NULL,
+				PRIMARY KEY (`id`)
+			);
+		</sql>
+	</changeSet>
+	<changeSet author="aldrin" id="v22.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="user_device_metadata"></tableExists>
+			</not>
+		</preConditions>
+		<comment>
+			Create table 'user_device_meta_data'
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+		  CREATE TABLE `workbench`.`user_device_meta_data` (
+			  `id`             INT NOT NULL AUTO_INCREMENT,
+			  `userid`         INT NULL,
```

Comment:
@abatac seems userid should not be null and have a FK to users

Reply:
Done.